conf/ocsci/downstream_config.yaml

```diff
@@ -10,4 +10,4 @@ ENV_DATA:
   # rook_csi_provisioner_image:
   # rook_csi_snapshotter_image:
   # rook_csi_attacher_image:
-  cluster_namespace: 'rook-ceph'
+  cluster_namespace: 'openshift-storage'
```

Comment:
"rook-ceph" was not a hack if I understand correctly, It was that we wanted to use a separate namespace and I believe "rook-ceph" is what comes as default, so it is worth leaving it as is.

Reply:
@vasu AFAIK, we would finally need 'openshift-storage' for proper integration with storage dashboard in web console. Many of the monitoring and other related tasks wont work if we use rook-ceph as namespace.

Reply:
Thanks! , I will go through the document, Resolving the comment for now